Weather Man/Weather_Man/FileReader.py

```diff
@@ -0,0 +1,59 @@
+class FileReader():
```

Comment:
Its called FileReader but it also stores data inside itself. So its not just a file reader then is it?

Reply:
FileExtractor